Update JS Release Notes (#1566)
SAP/cloud-sdk: docs-js/release-notes.mdx

<!-- vale off -->
<!-- This line is used for our release notes automation -->

## 3.7.0 [Core Modules] - October 17, 2023

### Fixed Issues

- [odata-common, odata-v2, odata-v4, generator, generator-common] Do not send `null` values in the request payload for properties that are not set while creating or updating an entity.

>   [Fixed Issue] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)

Reviewer: ```suggestion
 - [odata-common, odata-v2, odata-v4, generator, generator-common] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85) Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)
```

> - [odata-common, odata-v2, odata-v4, generator, generator-common] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85) Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)

Reviewer: ```suggestion
- [odata-common, odata-v2, odata-v4, generator, generator-common] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)
```

>  - [odata-common, odata-v2, odata-v4, generator, generator-common] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85) Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)

Reviewer: ```suggestion
- [odata-common, odata-v2, odata-v4, generator, generator-common] Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85) Fix `RangeError: Maximum call stack size exceeded` error that occurs when updating an entity after calling `getByKey`. (bdcf14f85)
```
